flocx_market/cmd/app.py

```diff
@@ -0,0 +1,11 @@
+from flocx_market.api import app
+from flocx_market.db.orm import orm
+from flask_migrate import Migrate
+
+application = app.create_app(app_name='flocx-market')
+migrate = Migrate(application, orm)
```

Comment:
This should be done in a separate script, something like cmd/dbsync.py. Something like https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/cmd/dbsync.py.  @DanNiESh can help if you have questions on that!